src/goto-programs/remove_calls_nobody.cpp

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************\
+
+Module: Remove calls to functions without a body
+
+Author: Daniel Poetzl
+
+\*******************************************************************/
+
+/// \file
+/// Remove calls to functions without a body
+
+#include "remove_calls_nobody.h"
+
+void remove_calls_nobodyt::remove_call_nobody(
```

Comment:
Instead of all methods being called `remove_call_nobody`, perhaps this could be called `generate_nondet_method_stub`